tests/ro-crate-metadata.json

```diff
@@ -0,0 +1,455 @@
+                },
+                {
+                    "@id": "#ERR034597_1.small_fastqc.html"
+                },
+                {
+                    "@id": "#ERR034597_2.small_fastqc.html"
+                }
+            ],
+            "programmingLanguage": {
+                "@id": "https://w3id.org/workflowhub/workflow-ro-crate#cwl"
+            },
+            "version": "1.0.0"
+        },
+        {
+            "@id": "https://w3id.org/cwl/v1.0/",
+            "@type": [
+                "WebPage"
+            ]
+        },
+        {
+            "@id": "https://www.commonwl.org/",
+            "@type": [
+                "WebPage"
+            ]
+        },
+        {
+            "@id": "https://w3id.org/workflowhub/workflow-ro-crate#cwl",
+            "@type": "ComputerLanguage",
+            "alternateName": "CWL",
+            "identifier": {
+                "@id": "https://w3id.org/cwl/v1.0/"
+            },
+            "name": "Common Workflow Language",
+            "url": {
+                "@id": "https://www.commonwl.org/"
+            },
+            "version": "1.0"
+        },
+        {
+            "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/README.md",
+            "@type": [
+                "TextObject"
+            ],
+            "name": "README.md"
+        },
+        {
+            "@id": "exe/fastqc.cwl",
+            "@type": [
+                "File",
+                "SoftwareSourceCode"
+            ],
+            "contentSize": 577,
+            "dateModified": "2023-01-13T09:13:29.886652",
+            "encodingFormat": "text/plain",
+            "lineCount": 33,
+            "sha512": "af6d23d05f8b42739feb4db4f3222e0201f7186953a4200bc3a72668589db672d738f27275eb5298cd2d4a1f5f3aa854f5f56a8561f5a8217a7f1c9663627b95",
+            "url": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/fastqc.cwl"
+        },
+        {
+            "@id": "exe/trimmomatic_pe.cwl",
+            "@type": [
+                "File",
+                "SoftwareSourceCode"
+            ],
+            "contentSize": 1570,
+            "dateModified": "2023-01-13T09:13:30.521735",
+            "encodingFormat": "text/plain",
+            "lineCount": 66,
+            "sha512": "cc384f32389e588706860139d843194bef48d53081183f12fc3ed32d3bc56bfca868c2965d5e9dd4e50a66e913b7b3afc0e0f9766353a14ad87f3b4a33b2ed2e",
+            "url": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimmomatic_pe.cwl"
+        },
+        {
+            "@id": "#26cee039-7a37-4647-924f-ee2187304ef3",
+            "@type": "CreateAction",
+            "actionStatus": [
+                {
+                    "@id": "CompletedActionStatus",
+                    "label": "COMPLETE",
+                    "value": 0
+                }
```

Comment:
This should be simply:

```json
"actionStatus": "CompletedActionStatus"
```

[https://schema.org/actionStatus](actionStatus) Takes [ActionStatusType](https://schema.org/ActionStatusType) values, which is an enumeration.  Moreover, RO-Crates have a flattened structure, so you couldn't put an object as the value of a property anyway.

Reply:
Fixed by https://github.com/sapporo-wes/sapporo-service/pull/32/commits/76f4b28351d30c836463fab644488a8b47fd0d3b